source/puppet/4.4/lang_type_aliases.md

```diff
@@ -1,82 +1,84 @@
 ---
 layout: default
 title: "Language: Type Aliases"
-canonical: "/puppet/latest/reference/lang_type_aliases.html"
+canonical: "/puppet/latest/lang_type_aliases.html"
 ---
 
 [reserved]: ./lang_reserved.html
 
+Type aliases allow you to create reusable and descriptive data and resource types.
 
-Type aliases allow you to create reusable and descriptive types.
+## Creating type aliases
 
-## Syntax
+Type aliases are written as `type <NAME> = <TYPE DEFINITION>`. The alias `<NAME>` begins with a capital letter and must not be a [reserved word][reserved].
 
-Type aliases are written as `type <NAME> = <TYPE DEFINITION>`, the alias `<NAME>` beginning with a capital letter. The alias name must not be a [reserved word][reserved]. 
+This example makes the type alias `MyType` equivalent to the `Integer` data type:
 
-As an example:
-
-~~~
+``` puppet
 type MyType = Integer
-~~~
+```
+
+You can then declare a parameter using the alias as though it were a unique type:
 
-Makes `MyType` equivalent to `Integer`.
+``` puppet
+MyType $example = 10
+```
 
-This mechanism is used for several reasons:
+By using type aliases, you can:
 
-* Gives a type a descriptive name such as `IPv6Addr`, rather than just using a complex pattern based type.
-* Shortens complex type expressions and moves them "out of the way".
-* Reuse of types increases the quality, as not everyone has to invent types like `IPv6Addr`.
-* Type definitions can be tested separately.
+-   Give a type a descriptive name, such as `IPv6Addr`, instead of creating or using a complex pattern-based type.
+-   Shorten and move complex type expressions.
+-   Improve code quality by reusing existing types instead of inventing new types.
+-   Test type definitions separately.
 
+### Type alias transparency
 
-## Location
+Puppet language aliases are transparent. This example's `notice` returns `true`:
 
-Store the type aliases you write in your modules' `types` folder, which is a top-level directory, a sibling of `manifests` and `lib`. Define only one alias per file.
+``` puppet
+type MyInteger = Integer
+notice MyInteger == Integer
+```
 
-The name of the `.pp` file defining the alias must be the alias name in all lower case without underscore separators inserted at camel case positions. For example, `MyType` is expected to be loaded from a file named `"mytype.pp"`.
+> **Note:** The internal types `TypeReference` and `TypeAlias` are never values in a Puppet program.
```

Comment:
Should instead be something to the effect of: "You should never use or alter `TypeReference` or `TypeAlias` in your implementation of Puppet, as they're internal types." 'Puppet program' is a meaningless henrikism